### PR TITLE
[All hosts] NAA timeline update

### PR DESCRIPTION
**Pull request by davidchesnut**
This is an update to the legacy Exchange token turn off dates, and NAA GA per channel.

### CODE REVIEW

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml [hunk 1]
```diff
@@ -20,28 +20,53 @@ sections:
 
       - question: What is the timeline for shutting down legacy tokens?
         answer: |
-          After October 2024, Outlook add-ins that rely on legacy Exchange tokens will not work as expected since Exchange will no longer issue those legacy tokens by default.
+          The following tables list the key milestones based on which channel customers are using. Note that the general availability (GA) date for NAA will vary based on channel. We'll provide tooling for administrators to reenable Exchange tokens for tenants and add-ins if those add-ins are not yet migrated to NAA.
```
Comment (AlexJerabek) [inline docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml:23]: Do we know how that tooling will be provided?

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml [hunk 2]
```diff
@@ -20,28 +20,53 @@ sections:
...
+          Unfortunately, no. We are following the timeline specified in the previous tables. We recognize that this migration is happening quickly, but we cannot offer exceptions to any partners. Migrating away from legacy tokens by the established timeline is a commitment to customers and their security teams. This effort will greatly improve protection of customers' vital email data.
       - question: Does my Outlook add-in have to migrate to NAA?
         answer: |
-          No, Outlook add-ins don't have to use NAA, although NAA offers the best authentication experience for users and best security posture for organizations. If add-ins are not using legacy Exchange tokens, they won't be affected by their deprecation. Add-ins using MSAL.js or other SSO methods that rely on Entra ID will still work after October 2024.
+          No, Outlook add-ins don't have to use NAA, although NAA offers the best authentication experience for users and best security posture for organizations. If add-ins are not using legacy Exchange tokens, they won't be affected by the deprecation of Exchange tokens. Add-ins using MSAL.js or other SSO methods that rely on Entra ID will continue to work.
```
Comment (AlexJerabek) [inline docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml:69]:
```suggestion
          No. Outlook add-ins don't have to use NAA, although NAA offers the best authentication experience for users and best security posture for organizations. If add-ins are not using legacy Exchange tokens, they won't be affected by the deprecation of Exchange tokens. Add-ins using MSAL.js or other SSO methods that rely on Entra ID will continue to work.
```

**Review by AlexJerabek — APPROVED** [review]

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml (hunk 1 above)
Comment (davidchesnut) [reply docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml:23]: That info is coming later...

File: docs/includes/legacy-exchange-token-deprecation.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
...
+> Legacy Exchange tokens are deprecated. Starting October 2024, legacy Exchange  [user identity](../outlook/authentication.md#exchange-user-identity-token) and [callback](../outlook/authentication.md#callback-tokens) tokens will begin to be turned off for Exchange Online tenants. For the timeline and details see our [FAQ page](../outlook/faq-nested-app-auth-outlook-legacy-tokens.md). This is part of [Microsoft's Secure Future Initiative](https://blogs.microsoft.com/on-the-issues/2023/11/02/secure-future-initiative-sfi-cybersecurity-cyberattacks/), which gives organizations the tools needed to respond to the current threat landscape. Exchange user identity tokens will still work for Exchange on-premises. Nested app authentication is the recommended approach for tokens going forward.
```
Comment (samantharamon) [inline docs/includes/legacy-exchange-token-deprecation.md:2]:
```suggestion
> Legacy Exchange tokens are deprecated. Starting October 2024, legacy Exchange  [user identity](../outlook/authentication.md#exchange-user-identity-token) and [callback](../outlook/authentication.md#callback-tokens) tokens will begin to be turned off for Exchange Online tenants. For the timeline and details, see our [FAQ page](../outlook/faq-nested-app-auth-outlook-legacy-tokens.md). This is part of [Microsoft's Secure Future Initiative](https://blogs.microsoft.com/on-the-issues/2023/11/02/secure-future-initiative-sfi-cybersecurity-cyberattacks/), which gives organizations the tools needed to respond to the current threat landscape. Exchange user identity tokens will still work for Exchange on-premises. Nested app authentication is the recommended approach for tokens going forward.
```

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml [hunk 4]
```diff
@@ -20,28 +20,53 @@ sections:
...
-      - question: Can my Outlook add-in get an exception to finish the migration after Oct 2024?
+      - question: Can my Outlook add-in get an exception to finish the migration later?
         answer: |
-          Unfortunately, no. We recognize that this migration is happening quickly, but we cannot offer exceptions to any partners. Migrating away from legacy tokens by October 2024 is a commitment to customers and their security teams. This effort will greatly improve protection of customers' vital email data.
+          Unfortunately, no. We are following the timeline specified in the previous tables. We recognize that this migration is happening quickly, but we cannot offer exceptions to any partners. Migrating away from legacy tokens by the established timeline is a commitment to customers and their security teams. This effort will greatly improve protection of customers' vital email data.
```
Comment (samantharamon) [inline docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml:66]:
```suggestion
          Unfortunately, no. We're following the timeline specified in the previous tables. We recognize that this migration is happening quickly, but we can't offer exceptions to any partners. Migrating away from legacy tokens by the established timeline is a commitment to customers and their security teams. This effort will greatly improve protection of customers' vital email data.
```

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml [hunk 5]
```diff
@@ -20,28 +20,53 @@ sections:
...
       - question: Does my Outlook add-in have to migrate to NAA?
         answer: |
-          No, Outlook add-ins don't have to use NAA, although NAA offers the best authentication experience for users and best security posture for organizations. If add-ins are not using legacy Exchange tokens, they won't be affected by their deprecation. Add-ins using MSAL.js or other SSO methods that rely on Entra ID will still work after October 2024.
+          No. Outlook add-ins don't have to use NAA, although NAA offers the best authentication experience for users and best security posture for organizations. If add-ins are not using legacy Exchange tokens, they won't be affected by the deprecation of Exchange tokens. Add-ins using MSAL.js or other SSO methods that rely on Entra ID will continue to work.
```
Comment (samantharamon) [inline docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml:69]:
```suggestion
          No. Outlook add-ins don't have to use NAA, although NAA offers the best authentication experience for users and best security posture for organizations. If add-ins aren't using legacy Exchange tokens, they won't be affected by the deprecation of Exchange tokens. Add-ins using MSAL.js or other SSO methods that rely on Entra ID will continue to work.
```

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml [hunk 6]
```diff
@@ -57,7 +82,7 @@ sections:
           If your add-in is for Exchange on-premises only (for example, Exchange 2019), it is not affected by this announcement.
       - question: What will happen to my Outlook add-ins if I don't migrate to NAA?
         answer: |
-          If you don't migrate your Outlook add-ins to NAA, they will stop working as expected in Exchange Online. In October 2024, Exchange Online will block legacy token issuance by default in all tenants. Any add-in that uses legacy tokens will not be able to access Exchange online resources.
+          If you don't migrate your Outlook add-ins to NAA, they will stop working as expected in Exchange Online. When Exchange tokens are turned off (according to the previous tables) Exchange Online will block legacy token issuance. Any add-in that uses legacy tokens will not be able to access Exchange online resources.
```
Comment (samantharamon) [inline docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml:85]:
```suggestion
          If you don't migrate your Outlook add-ins to NAA, they'll stop working as expected in Exchange Online. When Exchange tokens are turned off (according to the previous tables), Exchange Online will block legacy token issuance. Any add-in that uses legacy tokens won't be able to access Exchange online resources.
```

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml [hunk 7]
```diff
@@ -76,7 +101,7 @@ sections:
 
       - question: As an admin, I don't own an add-in that needs an update. What should I do?
         answer: |
-          If you rely on an independent software vendor (ISV) for your add-in, we recommend reaching out as soon as possible to confirm they have a plan and a timeline for moving off of legacy Exchange tokens. ISV developers should reach out directly to their Microsoft contacts with questions to ensure they are ready for the end of Exchange legacy tokens on October 2024.
+          If you rely on an independent software vendor (ISV) for your add-in, we recommend reaching out as soon as possible to confirm they have a plan and a timeline for moving off of legacy Exchange tokens. ISV developers should reach out directly to their Microsoft contacts with questions to ensure they are ready for the end of Exchange legacy tokens.
```
Comment (samantharamon) [inline docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml:104]:
```suggestion
          If you rely on an independent software vendor (ISV) for your add-in, we recommend reaching out as soon as possible to confirm they have a plan and a timeline for moving off of legacy Exchange tokens. ISV developers should reach out directly to their Microsoft contacts with questions to ensure they're ready for the end of Exchange legacy tokens.
```

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml (hunk 1 above)
Comment (samantharamon) [inline docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml:23]:
```suggestion
          The following tables list the key milestones based on which channel customers are using. Note that the general availability (GA) date for NAA will vary based on channel. We'll provide tooling for administrators to reenable Exchange tokens for tenants and add-ins if those add-ins aren't yet migrated to NAA.
```

File: docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml [hunk 8]
```diff
@@ -20,28 +20,53 @@ sections:
...
-          - Exchange 2019 and other on-premises versions of Exchange will continue to allow legacy Exchange user identity tokens and callback tokens.
   - name: Outlook add-in migration FAQ
     questions:
       - question: Why is Microsoft making Outlook add-ins migrate?
         answer: |
           Switching to Microsoft Graph using Entra ID tokens is a big improvement in security for Outlook and Exchange customers.
           Entra ID (formerly Azure Active Directory) is a leader in the identity and access management space. Customers can take advantage of zero trust features such as conditional access, MFA requirements, continual token monitoring, real time safety heuristics, and more that are not available with legacy Exchange tokens. Customers have much of their most important business data stored in Exchange, so it's vital that we ensure this data is protected. Migrating the whole Outlook ecosystem to using Entra ID tokens with Microsoft Graph greatly improves security for customer data.
```
Comment (samantharamon) [inline docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml:63]:
```suggestion
          Entra ID (formerly Azure Active Directory) is a leader in the identity and access management space. Customers can take advantage of zero trust features such as conditional access, MFA requirements, continual token monitoring, real time safety heuristics, and more that aren't available with legacy Exchange tokens. Customers have much of their most important business data stored in Exchange, so it's vital that we ensure this data is protected. Migrating the whole Outlook ecosystem to using Entra ID tokens with Microsoft Graph greatly improves security for customer data.
```

**Review by lindalu-MSFT — APPROVED** [review]
Not only is @samantharamon a top-notch writer, she's also a dang good technical editor! She beat me to it, and I didn't find a thing she missed.  She's got eagle :eyes:

**Pull request merged**